pkg/sfu/interceptor/rtx.go

```diff
@@ -0,0 +1,173 @@
+package interceptor
+
+import (
+	"strings"
+	"sync"
+
+	"github.com/pion/interceptor"
+	"github.com/pion/sdp/v3"
+	"github.com/pion/webrtc/v3"
+
+	"github.com/livekit/livekit-server/pkg/sfu/utils"
+	"github.com/livekit/protocol/logger"
+)
+
+const (
+	SDESRepairRTPStreamIDURI = "urn:ietf:params:rtp-hdrext:sdes:repaired-rtp-stream-id"
+
+	rtxProbeCount = 10
```

Comment:
is 10 probes enough? Looks like probe tries is decremented if either mid && rid || rsid is not empty or if not a padding packet. Seems like primary stream could exhaust it?

Reply:
The primary stream and rtx stream are two streams and will not share probe counts, so it is to probe rtx stream after exclude padding packet.